src/radix_connect/session_id.rs

```diff
@@ -0,0 +1,43 @@
+use crate::prelude::*;
+
+uniffi::custom_newtype!(SessionID, String);
+
+#[derive(
+    Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Ord, PartialOrd, Hash,
+)]
+pub struct SessionID(pub String);
```

Comment:
hmm what is the SessionID though? Is it a string? is it bytes, or is it a `UUID`?

Reply:
If it is a UUID, take a look at: `profile_id.rs` ! and use same pattern as that perhaps?

Reply:
it's uuid, I will update it